WickedEngine/wiGUI.h

```diff
@@ -291,6 +291,10 @@ namespace wi::gui
 		Widget();
 		virtual ~Widget() = default;
 
+    // Delete copy/move to prevent invalidating `this`
+    // since it is often captured by event handlers.
```

Comment:
You could add that It's not just that, but current Wicked Widgets also have some internal implementation that doesn't support being copied; otherwise it sounds like we're patronizing the programmers

Reply:
I don't think it was patronizing, but I changed it as requested. Plus, I undid that newline in NewContentBrowser.